docs/UserGuide.md

```diff
@@ -76,44 +48,76 @@ This guides aims to: <br>
 </div>
 
 **Highlights** <br>
-`commands` or `PARAMETERS`
+[`commands`](#glossary) or [`PARAMETERS`](#glossary) are inputs by users
 
 **Keyboard Inputs**<br>
 
 <button>enter</button>   <button>&uarr;</button>   <button>&darr;</button>
 
 <div style="page-break-after: always;"></div>
 
+## Quick start
+
+1. Ensure you have Java `11` or above installed in your Computer. You can refer to [this](#appendix-installing-java-11)
+
+2. Download the latest `PleaseHireUs.jar` [here](https://github.com/AY2223S1-CS2103T-W17-4/tp/releases).
+
+3. Copy the file to the folder you want to use as the _home folder_ for your application.
+
+4. Double-click the file to start the app. A [GUI](#glossary) similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
+   ![Ui](images/Ui.png)
+
+5. Type a [command](#glossary) in the [command box](#navigating-the-gui) and press <button>enter</button> to execute it. e.g. typing **`help`** and pressing <button>enter</button> will open the help window.<br>
+
+6. Some example commands you can try. You may refer to [this](#features) for more commands.
+
+* **`list`** : Lists all internships.
+
+* **`add n/Grab p/software engineer pr/ASSESSMENT web/https://www.grab.com/sg/about`** : Adds a new internship to the internship tracker.
+
+* **`delete 3`** : Deletes the 3rd internship shown in the current list.
+
+* **`clear`** : Deletes all internships.
+
+* **`exit`** : Exits the app.
+
+<div style="page-break-after: always;"></div>
+
 ## Navigating the GUI
 ![Ui](images/navigate_ui.png)
 
-**Command Box** <br>
+<h3 style="color:#008037;"> COMMAND BOX </h3>
+
 Enter your command here.
 
 <div markdown="block" class="alert alert-success">
 **:bulb: Tip:** <br>
 You can use <button>&uarr;</button> or <button>&darr;</button> to navigate through your command history
 </div>
 
-**Result Display** <br>
+<h3 style="color:#FF914D;"> RESULT DISPLAY </h3>
+
 Returns a feedback message after a command is executed.
 
-**Main Panel** <br>
-Displays the list of internships.
+<h3 style="color:#004AAD;"> MAIN PANEL </h3>
 
-**View Panel** <br>
-The panel displays the internship being viewed in greater detail. 
+Displays the list of internships.
 
 <div style="page-break-after: always;"></div>
 
-**Statistic Panel** <br>
+<h3 style="color:#C400FF;"> VIEW PANEL </h3>
+
+The panel displays the internship being viewed in greater detail.
+
+<h3 style="color:#7ED957;"> STATISTIC PANEL </h3>
+
 ![Bar Chart](images/BarChart.png)
 
 The Bar Chart (located in the bottom right corner of the application) displays the number of
 applications on every stage of the possible application process **which are currently listed by
 the application**.
```

Comment:
"which are currently listed in the main panel" I think more specific

Reply:
sorry i dont understand this part